src/main/java/org/truffleruby/core/VMPrimitiveNodes.java

```diff
@@ -155,6 +156,24 @@ protected boolean vmMethodIsBasic(RubyMethod method) {
 
     }
 
+    @Primitive(name = "vm_object_method_is_basic")
```

Comment:
I suggest `vm_builtin_method?` (our terminology) or `vm_basic_method?` (MRI terminology)

Reply:
I named it to match `vm_method_is_basic` - I would have made it a different specialisation of that if primitives supported optional arguments.

Reply:
I'll rename it then. `vm_method_is_basic` should probably also be renamed, but it's kind of out of scope of this PR.